### PR TITLE
ignore all node_module and bower_components directories

### PR DESCRIPTION
**Pull request by brandon93s**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ var homeOrTmp = require('home-or-tmp');
 
 var DEFAULT_IGNORE = [
 	'node_modules/**',
+	'**/node_modules/**',
```
Comment (sindresorhus) [inline index.js]: I think this makes the rule above moot.

File: index.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ var homeOrTmp = require('home-or-tmp');
...
 	'bower_components/**',
```
Comment (sindresorhus) [inline index.js]: Should also be done here.

**Comment by brandon93s** [comment]
Updated globs and squashed commits.

File: index.js (hunk 1 above)
Comment (Qix-) [inline index.js]: If it's a glob pattern, try `node_modules/**/*`.

Comment (sindresorhus) [inline index.js]: @Qix- That wouldn't match `foo/node_modules/`

Comment (Qix-) [inline index.js]: why wouldn't it?

Comment (brandon93s) [inline index.js]: Trailing `/**` matches all directories and subdirectories. `/**/*` matches all files in all directories and subdirectories. 

In this case, they're the same. The prior is simpler. The former would be to match a certain file type: 
`/**/*.js`

Comment (Qix-) [inline index.js]: Gotcha.

**Comment by sindresorhus** [comment]
Looks good. Thanks :)

**Pull request merged**